src/toolbar/toolbar-group.jsx

```diff
@@ -19,17 +18,20 @@ function getStyles(props, state) {
 
   const styles = {
     root: {
-      float,
       position: 'relative',
       marginLeft: firstChild ? -marginHorizontal : undefined,
       marginRight: lastChild ? -marginHorizontal : undefined,
+      display: 'flex',
+      justifyContent: 'space-between',
+      flexWrap: 'wrap',
```

Comment:
How come `flex-wrap` is enabled by default? Is there a specific reason?

I'd assume that 95% of the time, toolbars are going to be used with elements in a row. Correct me if I'm wrong though!